src/components/shared/Numeral/Numeral.tsx

```diff
@@ -29,7 +29,7 @@ const Numeral = ({
 
   return (
     <NumeralBase
-      prefix={prefix}
+      prefix={prefix && `${prefix} `}
```

Comment:
should we maybe just call the prefix with a space in front of it when we call this component instead? I know we sometimes don't want the prefix to have a space when displaying currency data and such

Reply:
For currency, it works as you mentioned. However, for the Numeral component, I decided to leave the space as is because it is used in many places, and we would need to check all of them in this case.